seqr/utils/middleware.py

```diff
@@ -33,7 +33,7 @@ class JsonErrorMiddleware(MiddlewareMixin):
     @staticmethod
     def process_exception(request, exception):
         if request.path.startswith('/api'):
-            exception_json = {'message': str(exception)}
+            exception_json = {'message': exception.args[0]}
```

Comment:
The reason for removing `str()` is: when the `exception` has unicode chars, if `exception` is a Python 2 `unicode` type, casting to Python 2 `str` will cause an error. Then, we have to import `str` from `future.builtins`, but it will cause another error if `exception` is a Python 2 `str`.

Reply:
I would rather in the very rare case we have an uncaught error with unicode the error handling doesn't behave, rather than all the time retuning worse error messages. I think we should include the unicode literals and futre string, and wrap this in a try - catch to handle python 2

Reply:
`try-except` maybe is better. I will try.